Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -270,25 +276,13 @@ open class BaseMapView: UIView, MapClient, MBMMetalViewProvider, CameraViewDeleg
     open override func awakeFromNib() {
         super.awakeFromNib()
 
-        guard let accessToken = BaseMapView.parseIBString(ibString: accessToken__) else {
-            fatalError("Must provide access token to the MapView in Interface Builder / Storyboard")
-        }
+        let mapInitOptions = mapInitOptionsDataSource?.mapInitOptions() as? MapInitOptions ??
```

Comment:
curious why we need a cast here…

Reply:
It's because `MapInitOptions` is a struct; the delegate method returns an `Any` which, amazingly, seems to work with Obj-C, otherwise the type would need to be an `NSObject`. It's a bit hacky, but I felt preferable over forcing the type to be an `NSObject`.